parse/EffectPythonParser.cpp

```diff
@@ -638,11 +638,11 @@ void RegisterGlobalsEffects(py::dict& globals) {
 
     // give_empire_unlockable_item_enum_grammar 
     for (const auto& uit : std::initializer_list<std::pair<const char*, UnlockableItemType>>{
-            {"UnlockBuildingType", UnlockableItemType::UIT_BUILDING},
-            {"UnlockShipPart", UnlockableItemType::UIT_SHIP_PART},
-            {"UnlockShipHull", UnlockableItemType::UIT_SHIP_HULL},
+            {"GiveEmpireBuilding", UnlockableItemType::UIT_BUILDING},
```

Comment:
that's a possibly misleading name... it doesn't actually give a building, it rather just makes it possible to produce the building

Reply:
May be then name it something like `MakeAvailable...`?

Reply:
Better, not great, but I don't have a perfect answer to suggest.

What "item types" conflict with `Unlock*` ?

Reply:
`Unlock*` are currently used for `UnlockableItemType` enum constants.